templates/new/config/config.exs

```diff
@@ -5,6 +5,10 @@
 # is restricted to this project.
 import Config
 
+# Starting nerves_bootstrap adds the required aliases to Mix.Project.config()
```

Comment:
How about something less technical since I don't think most Nerves users will understand this?

Perhaps:

```
# Enable the Nerves integration with Mix
```

Reply:
I like it. Updated.